src/uct/base/uct_mem.c

```diff
@@ -114,7 +104,7 @@ ucs_status_t uct_mem_alloc(void *addr, size_t min_length, unsigned flags,
 
                 /* Check if MD supports allocation with fixed address
                  * if it's requested */
-                if ((flags & UCT_MD_MEM_FLAG_FIXED) &&
+                if ((param->flags & UCT_MD_MEM_FLAG_FIXED) &&
```

Comment:
what if params->flags is not set (in multiple locations)

Reply:
As per documentation, uct_mem_alloc is for zcopy communication. Seems like UCT_MD_MEM_ACCESS_ALL or possibly UCT_MD_MEM_ACCESS_RMA is a minimum requirement for uct_mem_alloc. Should we error out if FIELD_FLAGS is not set?

Reply:
For now, I've added a check in params check to error out if PARAM_FIELD_FLAGS is not set. Lmk if we the condition is too strict.

Reply:
use local var for flags